examples/todo/components/TodoItemForm.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { Todo } from '../types'
+import { focusAtom, useSelector } from 'klyva'
+import { RemovableAtom, PrimitiveAtom } from 'klyva/dist/types'
+import { SmartTextInput } from './SmartTextInput'
+
+type TodoItemFormProps = {
+  todoAtom: RemovableAtom<Todo, any>
```

Comment:
Same as above comment:

> You can use `RemovablePrimitiveAtom` here so, or in place of `any` put `SetState<Todo> `

Reply:
Ah! I meant to ask you about that but forgot. Thank you, will fix!